sdk/identity/Azure.Identity/src/AuthenticationAccount.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client;
+
+namespace Azure.Identity
+{   internal class AuthenticationAccount : IAccount
+    {
+        private AuthenticationRecord _profile;
+
+        internal AuthenticationAccount(AuthenticationRecord profile)
+        {
+            _profile = profile;
+        }
+
+        string IAccount.Username => _profile.Username;
+
+        string IAccount.Environment => _profile.Authority;
+
+        AccountId IAccount.HomeAccountId => _profile.AccountId;
```

Comment:
I believe I originally was implementing this interface on a public type and implemented it explicitly to avoid the APIs being directly added to the type. But eventually I changed the design to make the class internal and just didn't change it to an implicit implementation.